packages/client/e2e/blotter.spec.ts

```
@@ -72,17 +72,45 @@ test.describe("Trade Blotter", () => {
   })
 
   test("when user buys a currency, the new row should flash briefly ", async () => {
+    const MAX_ATTEMPT = 2
+    let isNewRowFlashing = false
+
+    const placeTradeAndValidateFlashing = async () => {
+      try {
+        await tilePage.locator('[data-testid="Buy-EURUSD"]').nth(0).click()
+
+        const tradeID = await tilePage
+          .locator("[data-testid='trade-id']")
+          .innerText()
+
+        const newRow = blotterPage.getByTestId(`trades-grid-row-${tradeID}`)
+
+        await expect(newRow).toHaveCSS("animation", /1s ease-in-out/, {
+          timeout: Timeout.AGGRESSIVE,
```

Make the assertion fail faster as it should be flashing upon trade being placed